### ISSUE TITLE
Startup and anxiety recognition

### ISSUE THREAD

**Issue by sa-and** [gianlucatruda/sir2019-study-buddy-group-7#6]

**Comment by gianlucatruda** [comment]
- [x] We can do pretty rudimentary sentiment analysis using [TextBlob](https://textblob.readthedocs.io/en/dev/index.html#). 
- [ ] We have to figure out how exactly we want to quantify anxiety and if we also want to use certain "trigger words" in this assessment.

**Comment by gianlucatruda** [comment]
### This is how we can implement the sentiment analysis

```python
from textblob import TextBlob
import nltk

# Later on we'll need nltk for our Natural Language Processing with TextBlob.
# We also download the 'punkt' corpus.
nltk.download('punkt')
# This only needs to be done when you set up your environment, not every time we run

user_text = "whatever the user said, that we got from DialogFlow"
sent = TextBlob(user_text).sentiment
polarity = sent.polarity
subjectivity = sent.subjectivity
```

From [these docs](https://planspace.org/20150607-textblob_sentiment/), we can understand polarity roughly as "mood" and subjectivity as how much the statement is based on "opinion."

>This tells us that the English phrase “not a very great calculation” has a polarity of about -0.3, meaning it is slightly negative, and a subjectivity of about 0.6, meaning it is fairly subjective.

**Comment by gianlucatruda** [comment]
I've now implemented the above in PR #13
